superset-frontend/webpack.config.js

```diff
@@ -230,6 +230,7 @@ const config = {
       },
       {
         test: /\.tsx?$/,
+        exclude: [/\.test.tsx?$/],
```

Comment:
Maybe make this exclude `.test.jsx` too since there's a decent chance we might move files around before converting them to TS?

Reply:
Those are also excluded lower down, in the jsx (babel) loader.